service/src/test/kotlin/app/cash/backfila/actions/EditPartitionEndPointTest.kt

```diff
@@ -0,0 +1,162 @@
+package app.cash.backfila.actions
+
+import app.cash.backfila.BackfilaTestingModule
+import app.cash.backfila.api.ConfigureServiceAction
+import app.cash.backfila.client.Connectors
+import app.cash.backfila.dashboard.CreateBackfillAction
+import app.cash.backfila.dashboard.GetBackfillStatusAction
+import app.cash.backfila.dashboard.StartBackfillAction
+import app.cash.backfila.dashboard.StartBackfillRequest
+import app.cash.backfila.fakeCaller
+import app.cash.backfila.protos.service.ConfigureServiceRequest
+import app.cash.backfila.protos.service.CreateBackfillRequest
+import app.cash.backfila.service.persistence.BackfilaDb
+import app.cash.backfila.service.persistence.BackfillState
+import app.cash.backfila.service.persistence.RunPartitionQuery
+import com.google.inject.Module
+import javax.inject.Inject
+import misk.exceptions.BadRequestException
+import misk.hibernate.Query
+import misk.hibernate.Transacter
+import misk.hibernate.newQuery
+import misk.scope.ActionScope
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import okio.ByteString.Companion.encodeUtf8
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+
+@MiskTest(startService = true)
+class EditPartitionEndPointTest {
+  @Suppress("unused")
+  @MiskTestModule
+  val module: Module = BackfilaTestingModule()
+
+  @Inject lateinit var configureServiceAction: ConfigureServiceAction
+
+  @Inject lateinit var createBackfillAction: CreateBackfillAction
+
+  @Inject lateinit var getBackfillStatusAction: GetBackfillStatusAction
+
+  @Inject lateinit var startBackfillAction: StartBackfillAction
+
+  @Inject @BackfilaDb
+  lateinit var transacter: Transacter
+
+  @Inject lateinit var queryFactory: Query.Factory
+
+  @Inject lateinit var scope: ActionScope
+
+  /**
+   * Test-specific action that provides the same functionality as EditPartitionCursorHandlerAction
+   * but without the HTTP dependencies
+   */
+  inner class TestEditPartitionAction {
```

Comment:
So the test is only testing this fake action? That doesn't give us much. We should refactor the actual EditParitionCursorAction so it's testable, or test it as is handling the HTTP dependencies, or delete this test.